helm/default-apps-azure/values.yaml

```diff
@@ -101,6 +109,10 @@ apps:
     # used by renovate
     # repo: giantswarm/cert-manager-app
     version: 3.7.3
+    extraConfigs:
```

Comment:
I used extra config here since it is not possible to do a valid templating in values.yaml for this case. The priority is default (25) which means user config can overwrite it if necessary. CC: @nprokopic